pack-all.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I think it's not needed any more since latest release of BWC is 8.13.2, right?

Reply:
Deleted `pack-all` script